eng/build/Workers.Powershell.props

```diff
@@ -8,8 +8,9 @@
 
   <Target Name="RemovePowershellWorkerRuntimes" BeforeTargets="AssignTargetPaths" Condition="'$(RuntimeIdentifier)' != ''">
     <ItemGroup>
-      <_KeepPowerShellRuntime Include="win;win-x86;win10-x86;win-x64;win10-x64" Condition="$(RuntimeIdentifier.StartsWith(win))" />
-      <_KeepPowerShellRuntime Include="linux;linux-x64;unix" Condition="$(RuntimeIdentifier.StartsWith(linux))" />
+      <_KeepPowerShellRuntime Include="win;win-x86;win10-x86;win-x64;win10-x64;win-arm64" Condition="$(RuntimeIdentifier.StartsWith(win))" />
```

Comment:
I think we should include win-arm64 only when the RID is `win-arm64`.

Some context though: this file original was not specific about win-x86 vs win-x64 because that was how our previous `build-extension.ps1` script behaved. I wasn't brave enough to split it out and differentiate between them.

Reply:
Let me know what you think of the new conditions